_posts/2020-06-11-Solidity-0610-release-announcement.md

```diff
@@ -58,10 +58,9 @@ A notable new feature is the introduction of error codes. In Standard JSON input
 (``--standard-json``) the error codes will always be available under the property ``"errorCode"``.
 In the command-line interface, you can enable error codes via the switch ``--error-codes``.
 
-Two errors that contain a different error message (e.g. if the error concerns two
-different types) but are still essentially about the same thing will use the same error code.
+Similar errors (for example if error messages differ in type names only) might share the same error code. Vice versa, the same error message reported under different circumstances might have different error codes (although that is not a typical case).
 
-The error codes are intended to be stable across releases, but there might be some exceptions
+These error codes are intended to be reasonably stable across releases. However, there might be some exceptions, e.g. if one error is removed and another one added later, with the same error code.
```

Comment:
I think we can safely drop the second part - "reasonably" already captures that...

Reply:
Agree, I'll remove the second sentence.